extensions/blocks/instagram-gallery/edit.js

```diff
@@ -183,32 +239,17 @@ const InstagramGalleryEdit = props => {
 			{ showSidebar && (
 				<InspectorControls>
 					<PanelBody title={ __( 'Account Settings', 'jetpack' ) }>
-						<PanelRow>
-							<span>{ __( 'Account', 'jetpack' ) }</span>
+						<div>
+							{ __( 'Account:', 'jetpack' ) }{ ' ' }
 							<ExternalLink href={ `https://www.instagram.com/${ instagramUser }/` }>
 								@{ instagramUser }
 							</ExternalLink>
```

Comment:
I am not sure this is going to work on RTL languages, maybe this should be one translatable string with a placeholder.

Reply:
Ooh, makes sense, thanks!

Reply:
No, wait, now I remembered why I did it in two columns with `PanelRow` before: Guten `i18n` functions don't support React components.

I'll revert this change back to the two columns version. 🤦